styles/_search_results.scss

```
@@ -25,21 +92,74 @@
   &-list {
     margin-bottom: 25px;
     > li {
-      border-bottom: 1px solid $border-color;
       padding: 20px 0;
       &:first-child {
         border-top: 1px solid $border-color;
       }
+
+      h2 {
+        margin-bottom: 0;
+      }
     }
   }
-}
 
-.search-result {
+  .meta-group {
+    display: block;
+    @include desktop { display: flex; }
+    align-items: center;
+    clear: both;
+    color: #666;
+
+    > li {
+      display: block;
+      @include desktop { display: inline }
+    }
+
+    li:first-child {
+      flex: 1;
```

`flex` doesn't do anything for `display: block` children, so it makes it more apparent to scope this rule to desktop size.

```suggestion
      @include desktop {flex: 1 }
```